drink/src/runtime/minimal.rs

```diff
@@ -121,15 +124,9 @@ impl pallet_contracts::Config for MinimalRuntime {
     type MaxDebugBufferLen = ConstU32<{ 2 * 1024 * 1024 }>;
     type Migrations = ();
     type DefaultDepositLimit = DefaultDepositLimit;
-    type Debug = ();
+    type Debug = DrinkDebug;
```

Comment:
since runtime cannot be generic or customizable, I had to push the 'defaultness' down to `NoopDebugExt`

in other words:
 - I want to have just one runtime type in drink (`MinimalRuntime`); not counting things like `AlephRuntime` or `SubstrateContractsNodeRuntime` which actaully are just reexports
 - I want to give users choice if they want noop or custom logic

thus I have to put here `DrinkDebug` and push potential customization to extensions that are accessed by `DrinkDebug`

Reply:
Why keep the `NoopDebugExt` around then?